README.md

```diff
@@ -121,6 +122,18 @@ ReactDOM.createRoot(document.getElementById("root") as HTMLElement)
 
 ```
 
+### Custom Language Support

```

Comment:
cc @ryan-roemer I took your suggestion to bring this section to the forefront and verified the methods worked via our demo app